fastlane_bot/helpers/txhelpers.py

```diff
@@ -223,9 +223,14 @@ def _get_prices_info(
             / Decimal("10") ** Decimal("18")
         )
 
-        if self.ConfigObj.network.GAS_ORACLE_ADDRESS:
-            layer_one_gas_fee = self._get_layer_one_gas_fee_loop(signed_arb_tx)
-            gas_cost_eth += layer_one_gas_fee
+        if self.ConfigObj.NETWORK in ["optimism", "coinbase_base"]:
+            layer_one_gas_fee = self._get_layer_one_gas_fee_1(signed_arb_tx)
+        elif self.ConfigObj.NETWORK in ["mantle"]:
+            layer_one_gas_fee = self._get_layer_one_gas_fee_2(signed_arb_tx)
+        else:
+            layer_one_gas_fee = 0
+
+        gas_cost_eth += layer_one_gas_fee
```

Comment:
This change can be reverted, as it will be obtained directly from a similar task already completed for Base and Optimism (which would otherwise lead to manual conflict when this change is merged).